eng/targets/Settings.props

```diff
@@ -152,8 +146,6 @@
     <When Condition="'$(Language)' == 'VB'">
       <PropertyGroup>
         <LangVersion>15.5</LangVersion>
-        <RemoveIntegerChecks>true</RemoveIntegerChecks>
-        <RootNamespace></RootNamespace>
```

Comment:
Where is this set now?

Reply:
Setting `RootNamespace` to empty here is a no-op, since .NET Core SDK sets it to project name if empty.

RemoveIntegerChecks is set here: https://github.com/dotnet/roslyn/blob/master/eng/targets/RepoToolset/ProjectDefaults.props#L124

Reply:
Oh yes. I forgot that was the case and why we have to clear it on every single VB project.

Reply:
We don't necessarily need to. We could clear it in targets file, but when I did that some projects broke. I guess we can clean it up later.